sandman/source/schedule.cpp

```diff
@@ -223,10 +223,23 @@ static void ScheduleLogLoaded()
 			"up" : "down";
 			
 		// Print the event.
-		Logger::WriteFormattedLine("\t+%01ih %02im %02is -> %s, %s", delayHours, delayMin, 
-			delaySec, event.m_controlAction.m_controlName, actionText);
+		Logger::WriteLine(/* Set fill to `0`. */
```

Comment:
This comment is superfluous.

Reply:
I removed the comment that says "``/* Set fill to `0`. */``" in 1b6256e475da279e68dbb1eee4df2a6b25a0170e.